string command
---
bool success
float32 integral_force_x
float32 integral_force_y
float32 variance_force_z
